packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -120,6 +120,11 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
     this._initialFocusedIndex = props.initialFocusedIndex;
   }
 
+  public scrollToIndex(index: number, measureItem?: (itemIndex: number) => number): void {
+    this._list && this._list.scrollToIndex(index, measureItem);
+    this._groupedList && this._groupedList.scrollToIndex(index, measureItem);
+  }
```

Comment:
Can't we have a `componentRef` for `IList` on `DetailsList`? `listRef`?

Reply:
Not sure what you mean by `componentRef`. Can you provide an example?

Reply:
every component has a `componentRef` property that gives you the instance of the component. if you use plain `List` directly and you use `componentRef` property, you'll get an instance of `IList`. We could use a similar property to return `this._list || this._groupedList` to the user of `DetailsList`.. but now that I think more about it, what we have here might be fine, the underlying list is relatively transparent to the user of `DetailsList` and maybe we shouldn't leak that